prac_05/word_occurrences.py

```diff
@@ -0,0 +1,32 @@
+"""
+Word Occurrences
+Estimate: 35 minutes
+Actual:   26 minutes
+"""
+
+collection_of_words = {}
+
+sentence_of_words = input("Text: ")
+words = sentence_of_words.split()
+
+for word in words:
```

Comment:
for word in words:
        collection_of_words[word] = collection_of_words.get(word, 0) + 1
     
this can be a little simpler, but yours is nice for readability